lib/graph_ql/directives/directive_chain.rb

```diff
@@ -2,13 +2,13 @@ class GraphQL::DirectiveChain
   DIRECTIVE_ON = {
     GraphQL::Nodes::Field =>          GraphQL::Directive::ON_FIELD,
     GraphQL::Nodes::InlineFragment => GraphQL::Directive::ON_FRAGMENT,
-    GraphQL::Nodes::FragmentSpread => GraphQL::Directive::ON_FRAGMENT,
```

Comment:
I prefer "extra" commas because it makes cleaner diffs when things are added, removed or reordered